rules/exposure-to-internet-via-istio-ingress/rule.metadata.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "exposure-to-internet-via-istio-ingress",
+  "attributes": {
+    "useFromKubescapeVersion": "v3.0.9"
```

Comment:
so this means we don't have to release a new KS? only the upcoming chart with RBAC will suffice?

Reply:
It worked for me in the last CLI release, but I didn't know how to test it with the in-cluster Kubescape before release.